### ISSUE TITLE
Preview Route returns 404

### ISSUE THREAD

**Issue by acewebsa** [simonhamp/laravel-nova-csv-import#17]
Route::get('/preview/{file}', 'ImportController@preview');
is giving a 404 error.
If I do a dd just before the return line it is returned, so it is definitely finding the preview function without issue. Can't work out why this returns 404 though.
Any thoughts on why this might be?
Not sure if its related to #16 as I have a lot of resources with BelongsTo Fields in the project.

**Comment by fpcimmino** [comment]
I have the same problem

**Comment by acewebsa** [comment]
Yes you are right. How bizarre. I think I might look deeper for a fix.

**Comment by fpcimmino** [comment]
Ok, if you find the solution, write it here. I will do the same if I find it first. Thanks.

**Comment by acewebsa** [comment]
Cheers will do.

**Comment by acewebsa** [comment]
It's certainly related to #16 
When you take the BelongsTo out of $fields, it then succeeds.

**Comment by acewebsa** [comment]
Added this to the import controller, and it worked. 
Not real pretty though.
```
  foreach ($fields as $key => &$field)
        {
        	foreach ($field as $k => &$f)
	        {
	        	if (get_class($f) == 'Laravel\Nova\Fields\BelongsTo')
		        {
		        	unset($fields[$key][$k]);
		        }
	        }
        }
```

**Comment by fpcimmino** [comment]
Thanks for your help.
